### PR TITLE
Update tests to stub got rather than request

### PR DESCRIPTION
**Pull request by solidgoldpig**

### CODE REVIEW

File: lib/fb-submitter-client.unit.spec.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 const test = require('tape')
-const {stub, useFakeTimers} = require('sinon')
+const {stub, useFakeTimers} = require('sinon') // eslint-disable-line no-unused-vars
```
Comment (umaar) [inline lib/fb-submitter-client.unit.spec.js]: oh?

**Review by umaar — APPROVED** [review]

File: lib/fb-submitter-client.unit.spec.js (hunk 1 above)
Comment (solidgoldpig) [reply lib/fb-submitter-client.unit.spec.js]: er...

Comment (solidgoldpig) [reply lib/fb-submitter-client.unit.spec.js]: Fixed

File: lib/fb-submitter-client.unit.spec.js [hunk 2]
```diff
@@ -40,17 +40,17 @@ const expectedEncryptedData = 'pOXXs5YW9mUW1weBLNawiMRFdk6Hh92YBfGqmg8ych8PqnZ5l
  *
  **/
 const testInstantiation = (t, params, expectedCode, expectedMessage) => {
-  t.plan(4)
-
   let failedClient
   try {
     failedClient = new FBSubmitterClient(...params)
+    t.notOk('it should throw an error and not get here')
```
Comment (umaar) [inline lib/fb-submitter-client.unit.spec.js]: I think the semantics of '.throws()' is nicer but just a small preference

**Review by umaar — APPROVED** [review]

File: lib/fb-submitter-client.unit.spec.js (hunk 2 above)
Comment (solidgoldpig) [reply lib/fb-submitter-client.unit.spec.js]: ah yes - now I remember

Comment (solidgoldpig) [reply lib/fb-submitter-client.unit.spec.js]: Done

Comment (umaar) [reply lib/fb-submitter-client.unit.spec.js]: Thanks 👍

Comment (umaar) [reply lib/fb-submitter-client.unit.spec.js]: Just checked the code, think it should look more like this?

```
  const error = t.throws(() => fnWhichThrows(), 'should throw an error')
  t.equal(error.name, 'TheErrorName', '...')
  t.equal(error.data.thing, 'yes', '...')
```

As in is better to use one or the other, but not both

**Pull request merged**